who_are_we.md

```diff
@@ -0,0 +1,63 @@
+
+**Meticulous Execution**. This area is the core of how well we do things.  We implement tested processes to ensure the highest technical standards. For example, a complete deployment system is built into our projects, we use our own Gizra developed testing toolkit, and do pixel-perfect design implementation.
+
+**Investment in you and the community**. We can do better is about bettering ourselves and others as an organization and also as individuals.  You can become better by embracing our mentoring process that will usually keep you on your toes - being a little uncomfortable means you are stretching your bounds.  We also like seeing you want to contribute and encouraged you to do so to the open-source community on company time.
+
+Keep in mind and check yourself against these Gizra values. They will guide you in situations where you still don’t have an answer and set the standards you should want to reach.
+
+###Our Purpose:
+
+A general problem our industry faces is the high failure rate of web projects due to bad project management. The projects usually do not meet any one of the three criteria of success: being on time, in budget,happy users. 
+
+So, if we intersect the Gizra Way with what is going on outside, our purpose becomes clear:
+
+**For Gizra, the world will be a better place if more web projects realize that there is a better way.**
+
+###Our Position
+
+Our purpose probably strikes a chord with anyone who has implementing an IT project of any kind.  We want to reach the clients that can appreciate that there needs to be a better way.  Hence, we believe our message best reaches seasoned executive who have seen enough web projects to get burned. They may have already resigned to the idea that that is how projects are run but know deep down that there has to be another way. Our target clients are most probably in medium sized organizations but really it's about any place a brave executive can successfully influence their organization to adopt a new, simple, and sometimes radical, way of doing things.
+
+**For** medium sized companies
+
+**Who** have settled for mediocrity
+
+Gizra is a web solution provider
+
+**That** understands their client's needs and grows the project with them
+
+**Unlike** most web development companies
+
+Gizra’s budget driven approach with deep technical expertise results in better executed projects.
+
+###Our Personality
+
+Our unusual [About Story](http://www.gizra.com/content/the-about-story/) should get the point across pretty well.
+
+###Our Core Strengths
+
+We love doing projects that challenge us. This naturally led us to excel at the two edges of web development projects:
+
+**Complex Content Management Systems** Gizra was born from here. We joined the Drupal boat pretty early on and had the privilege to pioneered many of the core Drupal modules that serve complex client challenges.  Yes, we are experts here, and it's because we love solving problems.
```

Comment:
consistency with metaphor - either "join the community" or "board the boat" - I prefer the more traditional "join" here.